addons/l10n_au/data/template/account.tax-au.csv

```diff
@@ -54,20 +54,20 @@
 "","","","","","","","","","","","","base","refund","","-G10"
 "","","","","","","","","","","","","tax","refund","au_21330","-G10||-GST from General Ledger||Service"
 "au_tax_purchase_capital_service_tpar_no_abn","10% C TPAR N ABN","3","Capital Purchases","Capital Purchases","purchase","service","group","100.0","","tax_group_gst_10","au_tax_purchase_capital_service_tpar,au_tax_witheld","","","",""
-"au_tax_purchase_0_service","0% C","4","Capital Purchases","Capital Purchases","purchase","","percent","0.0","False","tax_group_gst_0","","base","invoice","","+G11||+G14"
+"au_tax_purchase_0_service","0% C","4","Zero Rated Purch","Zero Rated Purch","purchase","","percent","0.0","False","tax_group_gst_0","","base","invoice","","+G11||+G14"
 "","","","","","","","","","","","","tax","invoice","",""
 "","","","","","","","","","","","","base","refund","","-G11||-G14"
 "","","","","","","","","","","","","tax","refund","",""
-"au_tax_purchase_0_service_tpar","0% C TPAR","4","Capital Purchases","Capital Purchases","purchase","service","percent","0.0","False","tax_group_gst_0","","base","invoice","","+G11||+G14"
+"au_tax_purchase_0_service_tpar","0% C TPAR","4","Zero Rated Purch TPAR","Zero Rated Purch TPAR","purchase","service","percent","0.0","False","tax_group_gst_0","","base","invoice","","+G11||+G14"
 "","","","","","","","","","","","","tax","invoice","","l10n_au.service_tag"
 "","","","","","","","","","","","","base","refund","","-G11||-G14"
 "","","","","","","","","","","","","tax","refund","","l10n_au.service_tag"
-"au_tax_purchase_0_service_tpar_no_abn","0% C TPAR N ABN","4","Capital Purchases","Capital Purchases","purchase","service","group","100.0","","tax_group_gst_10","au_tax_purchase_0_service_tpar,au_tax_witheld","","","",""
-"au_tax_purchase_taxable_import_service","100% T EX","5","Purchase (Taxable Imports) - Tax Paid Separately","Purchase (Taxable Imports) - Tax Paid Separately","purchase","","percent","100.0","True","tax_group_gst_100000000","","base","invoice","","+G11||+G14"
+"au_tax_purchase_0_service_tpar_no_abn","0% C TPAR N ABN","4","Zero Rated Purch TPAR without ABN","Zero Rated Purch TPAR without ABN","purchase","service","group","100.0","","tax_group_gst_10","au_tax_purchase_0_service_tpar,au_tax_witheld","","","",""
+"au_tax_purchase_taxable_import_service","100% T EX","5","Purchase (Taxable Imports) - Tax Paid Separately","Purchase (Taxable Imports) - Tax Paid Separately","purchase","","division","100.0","True","tax_group_gst_100000000","","base","invoice","","+G11||+G14"

```

Comment:
The only place in the task that this is talked about says to not change anything.
May I know where this change comes from?

Reply:
They want to change it to this one at first but in the end still wrong. So they will keep looking for the correct one

Reply:
@vin-odoo Should we go ahead first or remove the update in this case?

Reply:
Better to not update something wrong by another wrong thing, at least it won't confuse peoples as to why it was updated this way

Reply:
Updated